presto-hive/src/main/java/com/facebook/presto/hive/rule/HiveFilterPushdown.java

```diff
@@ -294,6 +294,7 @@ public static ConnectorPushdownFilterResult pushdownFilter(
                                         remainingExpression,
                                         domainPredicate),
                                 currentLayoutHandle.map(layout -> ((HiveTableLayoutHandle) layout).getRequestedColumns()).orElse(Optional.empty()),
+                                false,
                                 false)),
```

Comment:
Should this be extracted from layout as well ?

Reply:
I am confused, appendRowNumber is always set to false.  but looking at the code above, requestedColumns is extracted from currentLayoutHandle, should the appendRowNumber be extracted from currentLayoutHandle as well ?

Reply:
Yes, updated it to inherit from currentLayoutHandle